sonic-xcvrd/xcvrd/xcvrd_utilities/port_event_helper.py

```diff
@@ -119,7 +130,7 @@ def handle_port_update_event(self):
                     (port_name, op, fvp) = port_tbl.pop()
                     if not port_name:
                         break
-                    if not validate_port(port_name):
+                    if not multi_asic.is_front_panel_port(port_name, self.port_role_map.get(port_name, None)):
```

Comment:
How will this logic work for dynamic ports? If there is DPB I don't think port_role_map gets updated

Reply:
Good catch!, Updated the logic to handle DPB